src/open_dread_rando/files/custom_scenario.lua

```diff
@@ -378,6 +391,8 @@ function Scenario.InitGui()
     if Init.bEnableRoomIds then
         RoomNameGui.Init()
     end
+
+    DisconnectGui.Init()
```

Comment:
We are initializing the UI even when we're not interested in it.

Reply:
It doesn't do any harm as the UI is not visible by default.

But I added a commit to not init the gui and also not schedule the function even once in no multiworld use case

Reply:
We are wasting memory though! We keep using more and more and I'm a bit scared of when we run out